app/models/registration_payment.rb

```diff
@@ -9,6 +9,7 @@ class RegistrationPayment < ApplicationRecord
   belongs_to :refunded_registration_payment, class_name: 'RegistrationPayment', optional: true
   has_many :refunding_registration_payments, class_name: 'RegistrationPayment', inverse_of: :refunded_registration_payment, foreign_key: :refunded_registration_payment_id, dependent: :destroy
 
+  after_create :auto_accept_hook
```

Comment:
Currently I'm performing a check for prod in the `attempt_auto_accept` method - but I could also do it here instead/as well

Reply:
Doing it here as well sounds like a good idea, you can never be sure enough.